src/main/java/com/wynntils/modules/map/overlays/ui/WorldMapUI.java

```diff
@@ -161,6 +162,26 @@ protected void createIcons() {
                 WorldMapIcon icon;
                 if (i instanceof MapLabel) {
                     icon = new WorldMapLabel((MapLabel) i);
+                } else if (i instanceof MapWaypointIcon) {
+                    icon = new WorldMapIcon(i);
+                    //Make sure this waypoint is named a Loot Chest, and uses the correct icons
+                    if (icon.getInfo().getName().startsWith("Loot Chest")) {
+                        WaypointProfile waypointProfile = ((MapWaypointIcon) i).getWaypointProfile();
+                        switch (waypointProfile.getType().getDisplayName()) {
+                            case "Chest (T1)":
+                                waypointProfile.setZoomNeeded(MapConfig.WorldMap.INSTANCE.lootChestTier1MinZoom);
+                                break;
+                            case "Chest (T2)":
+                                waypointProfile.setZoomNeeded(MapConfig.WorldMap.INSTANCE.lootChestTier2MinZoom);
+                                break;
+                            case "Chest (T3)":
+                                waypointProfile.setZoomNeeded(MapConfig.WorldMap.INSTANCE.lootChestTier3MinZoom);
+                                break;
+                            case "Chest (T4)":
+                                waypointProfile.setZoomNeeded(MapConfig.WorldMap.INSTANCE.lootChestTier4MinZoom);
+                                break;
+                        }
```

Comment:
Don't check for the display name, check literally for the enum values!
```suggestion
                        switch (waypointProfile.getType()) {
                            case LOOTCHEST_T1:
                                waypointProfile.setZoomNeeded(MapConfig.WorldMap.INSTANCE.lootChestTier1MinZoom);
                                break;
                            case LOOTCHEST_T2:
                                waypointProfile.setZoomNeeded(MapConfig.WorldMap.INSTANCE.lootChestTier2MinZoom);
                                break;
                            case LOOTCHEST_T3:
                                waypointProfile.setZoomNeeded(MapConfig.WorldMap.INSTANCE.lootChestTier3MinZoom);
                                break;
                            case LOOTCHEST_T4:
                                waypointProfile.setZoomNeeded(MapConfig.WorldMap.INSTANCE.lootChestTier4MinZoom);
                                break;
                        }
```